src/components/carousel/index.test.jsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from "@testing-library/react";
+
+import Carousel from ".";
+
+describe("Carousel", () => {
+	it("should render string child", () => {
```

Comment:
looks like there's no tests. is this ready for review or a wip?

Reply:
Working on tests.